app/src/main/res/layout/menu_blocking_switch.xml

```diff
@@ -2,17 +2,56 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<Switch xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The current menu item lets you click anywhere (ie. on the text instead of the switch), and it still toggles. IIUC the new version will only switch content blocking if you click on the switch, is that desired (I haven't tested it yet, so I'm not sure if the menu will just hide, without the content blocking setting changing, or if the menu will just stay open in this scenario?).